### ISSUE TITLE
Debian Hexchat crashes in freenode #bitcoin-otc (with/without plugins)

### ISSUE THREAD

**Issue by Veek** [hexchat/hexchat#1732]
It crashes soon as you move the mouse once you join the channel. I had reported a similar bug once earlier but this one, is in a different part of the program.

```
(hexchat:2752): GLib-GObject-WARNING **: invalid uninstantiatable type '<invalid>' in cast to '(null)'
(hexchat:2752): GLib-GObject-WARNING **: instance of invalid non-instantiatable type '<invalid>'
(hexchat:2752): GLib-GObject-CRITICAL **: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed
```

**Comment by TingPing** [comment]
https://wiki.debian.org/HowToGetABacktrace

**Comment by TingPing** [comment]
Also do try the latest release, 2.12.1 out; This issue has probably already been fixed.

**Comment by Veek** [comment]
hi, yeah it's fixed in 2.12 (i installed it from backports) - thanks
